static/js/src/advantage/react/components/Subscriptions/ExpiryNotification/ExpiryNotification.tsx

```diff
@@ -66,6 +66,8 @@ const MESSAGES: Messages = {
           "You have cancelled your Ubuntu Pro trial. " +
           "At the end of the trial period, this subscription " +
           "will disappear and you will no longer have access to Pro services.",
+        [UserSubscriptionType.Key_activated]:
+          "Contact OEM to renew and ensure service continuity.",
```

Comment:
Nitpick but, maybe `Contact the OEM` instead of `Contact OEM`? @leiaru-c

Reply:
Confirmed with Leia and fixed to `Contact the OEM`